src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_Hierarchy.cs

```diff
@@ -193,7 +195,7 @@ internal static bool IsOverride(Solution solution, ISymbol member, ISymbol symbo
         /// <returns>The derived types of the symbol. The symbol passed in is not included in this list.</returns>
 #pragma warning disable RS0026 // Do not add multiple public overloads with optional parameters
         public static async Task<IEnumerable<INamedTypeSymbol>> FindDerivedClassesAsync(
-            INamedTypeSymbol type, Solution solution, bool transitive, IImmutableSet<Project> projects = null, CancellationToken cancellationToken = default)
+            INamedTypeSymbol type, Solution solution, bool transitive = true, IImmutableSet<Project> projects = null, CancellationToken cancellationToken = default)
```

Comment:
pattern we decided on was to just have one entrypoint for all variants, just with an optional boolean flag stating what behavior is desired.